pkg/operator/ceph/object/realm/controller.go

```diff
@@ -210,6 +210,17 @@ func (r *ReconcileObjectRealm) reconcile(request reconcile.Request) (reconcile.R
 		}
 	}
 
+	// Set the realm as default if specified and supported
+	if cephObjectRealm.Spec.DefaultRealm {
+		objContext := object.NewContext(r.context, r.clusterInfo, cephObjectRealm.Namespace)
+		realmArg := fmt.Sprintf("--rgw-realm=%s", cephObjectRealm.Name)
+		_, err := object.RunAdminCommandNoMultisite(objContext, false, "realm", "default", realmArg)
```

Comment:
Can this code path call the `setDefaultRealm()` method in the object controller? The command looks the same? It would be good to avoid the duplication in implementation.